src/components/Sidebar.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { RiPlayListAddLine } from "react-icons/ri";
+import { AiFillCloseCircle } from "react-icons/ai";
+import { useRouteMatch, Link } from "react-router-dom";
+
+function StackList({ name }) {
+  let { url } = useRouteMatch();
+
+  return (
+    <Link className="link stacklist" to={`${url}/${name}`}>
+      <h1>{name}</h1>
+    </Link>
```

Comment:
Just try to be consistent with the spacing .Overall  great job ! Keep going.

Reply:
Thanks, fixed it